Uplift/Controllers/OnboardingGymsViewController.swift

```diff
@@ -92,25 +113,8 @@ class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITab
         backButtonArrow.alpha = 0.5
         backButton.addSubview(backButtonArrow)
         view.addSubview(backButton)
-        
-        toggleButton(button: nextButton, arrow: nextButtonArrow, enabled: false)
-
-        NetworkManager.shared.getGyms { (gyms) in
-            var namesArray = [String]()
-            gyms.forEach {
-                namesArray.append($0.name)
-            }
-            DispatchQueue.main.async {
-                self.gymNames = namesArray.sorted { $0 < $1 }
-                self.gymsTableView.reloadData()
-            }
-        }
-
-        let edgeSwipe = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(goBackAView))
-        edgeSwipe.edges = .left
-        view.addGestureRecognizer(edgeSwipe)
 
-        setUpConstraints()
+        toggleButton(button: nextButton, arrow: nextButtonArrow, enabled: false)
```

Comment:
Y r we passing in button and arrow when they're stored on the VC?

Reply:
good question